pkg/grpc/exec/exec.go

```diff
@@ -288,6 +288,36 @@ func GetProcessExit(event *MsgExitEventUnix) *tetragon.ProcessExit {
 	code := event.Info.Code >> 8
 	signal := readerexec.Signal(event.Info.Code & 0xFF)
 
+	// Now since we want to correlate the exit and exec events, then
+	// the final TID of the exit event _must matches_ the PID and TID of exec
+	// event. The tetragonEvent.Process of this exit event is constructed either:
+	//   1. When looking up the process by its PID from user space cache
+	//      then we get the process that was pushed during exec or clone.
+	//   2. Out of order exec event, and this is same as (1).
```

Comment:
I don't understand the (2) bullet here. I think it can just be dropped? If exec/exit are received OOO we we have to hold the exit event until we process the exec event. But its not really different than 1 which I think is what this comment says. Either way I found 2. more confusing than helpful. Maybe other have opinion.

Reply:
Indeed just trying to say that (2) will end up same as (1) but it takes different code path, maybe you are too familiar with it ;-) , will drop it sure

Reply:
ok removed